python/CHANGELOG.rst

```diff
@@ -19,6 +19,11 @@
 - Add ``TreeSequence.mutations_edge`` which returns the edge ID for each mutation's
```

Comment:
Let's add a big warning here that we're moving to numpy 2.0 by default, along with the errors you might get and what to do. For example, what happens when you have older msprime, kastore etc lying around that are compiled againsd numpy 1.0?

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/3228/commits/13a08a0bd9df82b01ffb09918fe5d35b90d84c21